modules/tinymce/src/core/test/ts/browser/content/EditorContentNotInitializedTest.ts

```diff
@@ -17,11 +16,6 @@ describe('browser.tinymce.core.content.EditorContentNotInitializedTest', () => {
 
   const createEditor = () => new Editor('editor', settings, EditorManager);
 
-  const setContentAndAssertReturn = (editor: Editor, content: AstNode | string) => {
-    const actual = editor.setContent(content);
-    assert.deepEqual(actual, content, 'should return what you tried to set');
-  };
```

Comment:
wouldn't be better to change this in:
```
  const setContentAndAssertReturn = (editor: Editor, content: AstNode | string) => {
    editor.setContent(content);
    const actual = editor.getContent();
    assert.deepEqual(actual, content, 'should return what you tried to set');
  };
```
using only `setContent` in these tests seems not to test anything :/

Reply:
It tests that it doesn't throw. But maybe I should wrap that so it's more apperent that it's what this is testing now.